specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -186,6 +186,137 @@
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "A successful call results with an Operation-Location header used to check the status of the analysis job.",
+            "headers": {
+              "Operation-Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Transcript Conversation Analysis Job Request": {
+            "$ref": "./examples/SuccessfulAnalyzeConversationTranscriptJobsTaskSubmitRequest.json"
+          },
+          "Successful Text Conversation Analysis Job Request": {
+            "$ref": "./examples/SuccessfulAnalyzeConversationTextJobsTaskSubmitRequest.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/analyze-conversation/jobs/{jobId}": {
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "description": "Get the status of an analysis job.  A job may consist of one or more tasks.  Once all tasks are succeeded, the job will transition to the succeeded state and results will be available for each task.",
+        "operationId": "AnalyzeConversation_JobStatus",
+        "summary": "Get analysis status and results",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/JobId"
+          },
+          {
+            "$ref": "#/parameters/ShowStats"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Analysis job status and metadata.",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeConversationJobState"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Transcript Conversation Analysis Job Status Request": {
+            "$ref": "./examples/SuccessfulAnalyzeConversationTranscriptJobsTaskStatusRequest.json"
+          },
+          "Successful Get Text Conversation Analysis Job Status Request": {
+            "$ref": "./examples/SuccessfulAnalyzeConversationTextJobsTaskStatusRequest.json"
+          }
+        }
+      }
+    },
+    "/analyze-conversation/jobs/{jobId}:cancel": {
+      "post": {
+        "produces": [
+          "application/json"
+        ],
+        "description": "Cancel a long-running analysis job on conversation.",
+        "operationId": "AnalyzeConversation_CancelJob",
+        "summary": "Cancel a long-running analysis job on conversation.",
+        "parameters": [
```

Comment:
We need to add the API version parameter here as well.

Reply:
will add